site/content/docs/main/restore-resource-modifiers.md

```diff
@@ -25,22 +25,22 @@ Below is the two-step of using resource modifiers to modify the resources during
 **YAML template**
 
 - Yaml template:
-    ```yaml
-   version: v1
-   resourceModifierRules:
-   - conditions:
-      groupKind: persistentvolumeclaims
-      resourceNameRegex: "^mysql.*$"
-      namespaces:
-      - bar
-      - foo
-   patches:
-   - operation: replace
-      path: "/spec/storageClassName"
-      value: "premium"
-   - operation: remove
-      path: "/metadata/labels/test"
-    ```
+ ```yaml
+version: v1
+resourceModifierRules:
+- conditions:
+     groupKind: persistentvolumeclaims
+     resourceNameRegex: "^mysql.*$"
+     namespaces:
+     - bar
+     - foo
+  patches:
```

Comment:
Seems the indent of patches section is still incorrect when you view the file as Markdown.

Maybe it's related to the space before "```yaml"?

Reply:
fixed it again.